src/app/types.ts

```diff
@@ -16,7 +16,7 @@ export type JarStatisticRecord = {
   id: number;
```

Comment:
While you here, pls also fix other cases with `snake_case`, like `owner_name` in `Jar`, etc.